### ISSUE TITLE
Update Guzzle to Version 7

### ISSUE THREAD

**Issue by jbeales** [php-opencloud/openstack#312]
Selfish request, but Guzzle 7 will make it so I can use this in a Laravel 8 app much more easily.

(Working on a PR now).

**Comment by andrecolza** [comment]
I need it too!

**Comment by jbeales** [comment]
I added my fork as a vcs repository to my composer.json, and tagged a 3.0.10 "release" on the Guzzle 7 branch, so now my composer install uses that and I was able to do the Laravel 8 update.

It's kind of hacky, and I'll probably have to delete my vendors directory & composer.lock and do a fresh composer install when the official release comes out, but it allows me to keep working.  Also, when it gets deployed it'll let the Guzzle 7 version of this package get a little production testing, (only for object storage, because that's all I'm doing with it at the moment).
